src/main/java/frc/robot/HardwareMap.java

```diff
@@ -15,4 +15,5 @@
 	// Pneumatics channel numbers
 	public static final int PCM_CHANNEL_INTAKE_CYLINDER_FORWARD = 1;
```

Comment:
Delete old hardware mappings that we're not using. Also, It would be great to keep the naming prefix `PCM_CHANNEL_*` to make it clear this is a PCM channel number.

Reply:
Renamed with new prefix, and redundant hardware mappings are removed